helpers/setup-spark-k8s.py

```diff
@@ -137,5 +188,21 @@ def setup_spark_conf_defaults(username: str, namespace: str) -> None:
     if args.action == 'service-account':
         username = args.username
         namespace = args.namespace or defaults['namespace']
-        set_up_user(username, namespace, defaults)
-        setup_spark_conf_defaults(username, namespace)
+        set_up_user(username, namespace, defaults, args.primary)
+        utils.setup_kubernetes_secret_literal(username, namespace, args.context, args.conf)
+    elif args.action == 'service-account-cleanup':
+        cleanup_user(args.username, args.namespace, args.context)
+    elif args.action == 'sa-conf-put':
+        utils.setup_kubernetes_secret_env_file(args.username, args.namespace or defaults['namespace'], args.context, args.properties_file)
+    elif args.action == 'sa-conf-get':
+        conf = utils.retrieve_kubernetes_secret(args.username, args.namespace, args.context, args.conf)
```

Comment:
why having two different UX for setting up account and sa-conf-put?

I would just allow both to take either --conf or --properties-file (with --conf overriding --properties-file if the same key is provided). It would be good to have the same function in both places